tests/bot/exts/moderation/test_silence.py

```diff
@@ -16,20 +16,19 @@
 )
 
 redis_session = None
-redis_loop = asyncio.get_event_loop()
 
 
 def setUpModule():  # noqa: N802
     """Create and connect to the fakeredis session."""
     global redis_session
     redis_session = RedisSession(use_fakeredis=True)
-    redis_loop.run_until_complete(redis_session.connect())
+    asyncio.run(redis_session.connect())
```

Comment:
This seems wrong because `asyncio.run` creates a new event loop. Presumably this will be separate from the event loop the rest of the tests use, unless unittest accounts for that somehow.

Reply:
There isn't a running loop when `redis_loop` was created, nor is there one during module setup, so the `asyncio.get_event_loop()` call was raising a deprecation error.

I think the alternative to this is to create the session within each test class as an `asyncSetUp`, rather than at the module level.

Reply:
I've pushed both [`1b6b76f` (#2229)](https://github.com/python-discord/bot/pull/2229/commits/1b6b76f3993d7307667d002722c1956fe2dd7a02) and [`b5797ff` (#2229)](https://github.com/python-discord/bot/pull/2229/commits/b5797ffcc2cd9a3efc0ae9c378418c5af17870e3) to show what this alternative would look like.